core-sitemaps.php

```diff
@@ -25,51 +25,56 @@
  * Version:           0.3.0
  */
 
+// Bail early if Sitemaps is already defined. Prevents plugin from loading after core merge.
+if ( class_exists( 'Sitemaps' ) ) {
+	return;
+}
+
 // The limit for how many sitemaps to include in an index.
-const CORE_SITEMAPS_MAX_SITEMAPS    = 50000;
-const CORE_SITEMAPS_REWRITE_VERSION = '2020-04-29';
+const SITEMAPS_MAX_SITEMAPS    = 50000;
+const SITEMAPS_REWRITE_VERSION = '2020-04-29';
 
 // Limit the number of URLs included in a sitemap.
-if ( ! defined( 'CORE_SITEMAPS_MAX_URLS' ) ) {
-	define( 'CORE_SITEMAPS_MAX_URLS', 2000 );
+if ( ! defined( 'SITEMAPS_MAX_URLS' ) ) {
+	define( 'SITEMAPS_MAX_URLS', 2000 );
 }
 
-require_once __DIR__ . '/inc/class-core-sitemaps.php';
-require_once __DIR__ . '/inc/class-core-sitemaps-provider.php';
-require_once __DIR__ . '/inc/class-core-sitemaps-index.php';
-require_once __DIR__ . '/inc/class-core-sitemaps-registry.php';
-require_once __DIR__ . '/inc/class-core-sitemaps-renderer.php';
-require_once __DIR__ . '/inc/class-core-sitemaps-stylesheet.php';
-require_once __DIR__ . '/inc/providers/class-core-sitemaps-posts.php';
-require_once __DIR__ . '/inc/providers/class-core-sitemaps-taxonomies.php';
-require_once __DIR__ . '/inc/providers/class-core-sitemaps-users.php';
+require_once __DIR__ . '/inc/class-sitemaps.php';
+require_once __DIR__ . '/inc/class-sitemaps-provider.php';
+require_once __DIR__ . '/inc/class-sitemaps-index.php';
+require_once __DIR__ . '/inc/class-sitemaps-registry.php';
+require_once __DIR__ . '/inc/class-sitemaps-renderer.php';
+require_once __DIR__ . '/inc/class-sitemaps-stylesheet.php';
+require_once __DIR__ . '/inc/providers/class-sitemaps-posts.php';
+require_once __DIR__ . '/inc/providers/class-sitemaps-taxonomies.php';
+require_once __DIR__ . '/inc/providers/class-sitemaps-users.php';
 require_once __DIR__ . '/inc/functions.php';
 
 // Boot the sitemaps system.
-add_action( 'init', 'core_sitemaps_get_server' );
+add_action( 'init', 'wp_sitemaps_get_server' );
 
 /**
  * Plugin activation hook.
  *
  * Adds and flushes rewrite rules.
  */
-function core_sitemaps_plugin_activation() {
-	$core_sitemaps = new Core_Sitemaps();
-	$core_sitemaps->register_rewrites();
+function sitemaps_plugin_activation() {
```

Comment:
See above, functions should also start with `wp_sitemaps_*`.

Reply:
ok

Reply:
these plugin activation/deactivation functions will not be part of core.